util/encryption.go

```
@@ -36,6 +44,42 @@ func InitializeEncryption() {
 	keyPresent = true
 }
 
+func setDefaultEncryptionKey() error {
+	if os.Getenv("SOURCES_ENV") != "stage" && os.Getenv("SOURCES_ENV") != "prod" {
+		_, filename, _, ok := runtime.Caller(1)
+		if !ok {
+			return fmt.Errorf("Not possible to recover the information")
+		}
+
+		filepath := path.Join(path.Dir(filename), "encryption_key.dev")
+		symError := os.Symlink(filepath, "example.dev")
+		if symError != nil {
+			return fmt.Errorf("Error within symlink")
+		}
+		if _, err := os.Lstat("example.dev"); err == nil {
+			if err := os.Remove("example.dev"); err != nil {
+				return fmt.Errorf("failed to unlink: %v", err)
+			}
+		} else if os.IsNotExist(err) {
+			return fmt.Errorf("failed to check symlink: %v", err)
+		}
+
```

Ohhhhh I see why you needed it now. 

I should have maybe been more specific. We don't want to make the symlink for the user in code - we want to just return a nice error that says "hey you need to symlink `encryption_key.dev` to `encryption_key`". 

That way the user themselves have to run `ln -s ...` we could even put the command in the error message.